README.md

```diff
@@ -1,2 +1,126 @@
-# hello-microverse
-Microverse first project
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="images/predator2.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Hello Microverse</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [⭐️ Show your support](#support)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello Microverse <a name="about-project"></a>
+
+**Hello Microverse** is the first project in Microverse Bootcamp.  
+In this project, we have created a simple HTML and CSS page using linters and github workflow.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML</summary>
+</details>
+
+<details>
+  <summary>CSS</summary>
+</details>
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
```

Comment:
- [ ] You constructed your `README.md` file well for a first-timer. That is impressive 👏. Kindly add the _**Key Features**_ section that should contain 1 - 3 features that were implemented in this project. You would need to add this section for all projects you undertake in Microverse and the professional space because it is a section that gives details about what we should look out for when we clone a project. 👍 😎

Reply:
I'm sorry dear reviewer, I didn't know what to add as a key features, if you would give me some examples i will grate you so much.

Reply:
> I'm sorry dear reviewer, I didn't know what to add as a key features, but I added some key features. if you would give me some examples about key features I will grate you so much.

Hi, sorry for the late response. I see you have added some features. Great job 👏